test/Integration/tpp-transpose.mlir

```diff
@@ -0,0 +1,15 @@
+// RUN: tpp-run --vector-To-XSMM %s -e entry -entry-point-result=void -print --seed 123 2>&1 | FileCheck %s
+// RUN: tpp-run --linalg-to-loops %s -e entry -entry-point-result=void -print --seed 123 2>&1 | FileCheck %s
+
+module {
+  func.func @entry(%arg0: tensor<3x5xf32>, %arg1: tensor<5x3xf32>)->tensor<5x3xf32> {
+    %out = linalg.transpose ins(%arg0 : tensor<3x5xf32>) outs(%arg1 : tensor<5x3xf32>) permutation = [1, 0]
```

Comment:
From my interpretation, your pattern matching above needs vector read/write to match the pattern. Will this really be converted into XSMM calls?

Can you add another RUN like that prints the MLIR=late and CHECK for xsmm calls?

Reply:
Will add print-mid/print-late

Reply:
Done.